src/ServiceManagement/RemoteApp/Commands.RemoteApp.Tests/Collection/RemoteAppCollectionSessionsTests.cs

```diff
@@ -0,0 +1,124 @@
+namespace Microsoft.Azure.Commands.Test.RemoteApp
+{
+    using Common;
+    using Microsoft.Azure.Management.RemoteApp;
+    using Microsoft.Azure.Management.RemoteApp.Cmdlets;
+    using Microsoft.Azure.Management.RemoteApp.Fakes;
+    using Microsoft.Azure.Management.RemoteApp.Models;
+    using Microsoft.QualityTools.Testing.Fakes;
+    using Moq;
+    using System;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using System.Net;
+    using VisualStudio.TestTools.UnitTesting;
+
+    [TestClass]
+    public class RemoteAppCollectionSessionsTests : RemoteAppClient
+    {
+        [TestMethod]
+        public void CanGetNoSessionsFromCollectionWithNoSessions()
+        {
+            String collection0SessionName = "testcollection";
+            String requestId = Guid.NewGuid().ToString();
+            DateTime logonTimeUtc = DateTime.UtcNow;
+
+            Mock<ICommandRuntime> mockRuntime = new Mock<ICommandRuntime>();
+
+////            Mock<RemoteAppManagementClient> mockClient = new Mock<RemoteAppManagementClient>();
```

Comment:
this file does not seems to have any valid test. Please remove it completely or uncomment tests.